op.c

```diff
@@ -1663,7 +1663,7 @@ Perl_alloc_LOGOP(pTHX_ I32 type, OP *first, OP* other)
 =for apidoc op_contextualize
 
 Applies a syntactic context to an op tree representing an expression.
-C<o> is the op tree, and C<context> must be C<G_SCALAR>, C<G_ARRAY>,
+C<o> is the op tree, and C<context> must be C<G_SCALAR>, C<G_LIST>,
```

Comment:
Again, will this cause folks to write code that doesn't easily port back to previous perls, or is Devel::PPPort supposed to solve that?

Reply:
It's a trivial oneliner in ppport.h:

```c
#define  G_LIST  G_ARRAY
```